scripts/common-service.Makefile

```diff
@@ -100,8 +106,7 @@ TEST_TARGET := $(if $(target),$(target),$(CURDIR)/tests/unit)
 
 _run-test-dev: _check_venv_active
 	# runs tests for development (e.g w/ pdb)
-	pytest -vv --exitfirst --failed-first --durations=10 --pdb --color=yes \
-	--cov=$(APP_PACKAGE_NAME) --cov-report=term-missing --cov-config=.coveragerc $(TEST_TARGET)
+	pytest -vv --exitfirst --failed-first --durations=10 --pdb --color=yes --cov=$(APP_PACKAGE_NAME) --cov-report=term-missing --cov-config=.coveragerc $(TEST_TARGET)
```

Comment:
not sure why you have that on a very long line now??

Reply:
mhh ... me neither ... , I guess i was comparing